history/characters/42000_high_elf.txt

```diff
@@ -53,24 +56,44 @@
 	properties="ai0e0a00000000"
 	culture=high_elf religion=cult_of_sunwell
 	martial=7 diplomacy=6 stewardship=6 intrigue=7 learning=8
-	add_trait=mastermind_theologian add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=wroth add_trait=cynical
+	add_trait=mastermind_theologian
+	add_trait=content # Don't want to rule the elves
+	add_trait=just # Though he is not worthy to be the next king
```

Comment:
I suggest to change comment or trait here, because "though he is not worthy to be the next king" is *just* not sounds like *justification* for just trait at all.

Reply:
Why? CK2 describes the trait...
> The character has a well-developed sense of justice

And following his sense of justice, he thought he's not _justified_ to be a king.

Reply:
> Why? CK2 describes the trait...
> > The character has a well-developed sense of justice

Ok, then let's find "sense of justice" definition

> A sense of justice consists of thoughts and feelings about what is fair and unfair
and what people deserve from and owe others (rights and duties).

Next:

> And following his sense of justice, he thought he's not justified to be a king

Well, let's *just* say that we can't surely tell what character think, unless we ask the author who create his personality, (send me original where you found this `Though he is not worthy to be the next king` thing, btw, since I think you may have interpret his words wrong) so let's *just* stick to your argumentation.

As I understood, your argument is that Kael'thas' has a well-developed sense of justice, that tells him that he, the crown prince, is not worthy to become the next king.

Now, look again at definition, since I need to ask the following questions:
- Why is the crown prince do not deserve to become a king?
- Why is it unfair that crown prince will become a king?

By whatever inheritance laws elves have, it's *justified* for the crown prince to become a king after death of its predecessor, by the right of birth. If Kael'thas really think that he is `not worthy to be the next king` and `don't want to rule the elves`, then he neither known his right, nor duties, and it means don't has "a well-developed sense of justice", isn't it?

Reply:
> Well, let's _just_ say that we can't surely tell what character think, unless we ask the author who create his personality, (send me original where you found this `Though he is not worthy to be the next king` thing, btw, since I think you may have interpret his words wrong) so let's _just_ stick to your argumentation.

https://wow.gamepedia.com/Kael%27thas_Sunstrider#Third_War

> Kael performed a Thalassian blessing and told his father that he was lost without him; that Anasterian was the king, not him.

> Kael'thas made no excuses for being absent in Quel'Thalas's darkest hour. He resolved to prove himself to the elves and do whatever he could to rebuild their kingdom. It was hard as he had always felt distant from his people. They were largely insular and isolationist and he was quite the opposite. He wanted to see the world, interact with other races and learn new ideas. Despite this, he loved his people and kingdom.

> Kael'thas officially decreed that his father would be the last king of Quel'Thalas, leaving the crown unclaimed.

> Now, look again at definition, since I need to ask the following questions:
> 
> * Why is the crown prince do not deserve to become a king?
> * Why is it unfair that crown prince will become a king?
> 
> By whatever inheritance laws elves have, it's _justified_ for the crown prince to become a king after death of its predecessor, by the right of birth. If Kael'thas really think that he is `not worthy to be the next king` and `don't want to rule the elves`, then he neither known his right, nor duties, and it means don't has "a well-developed sense of justice", isn't it?

Emm, it's a sense of justice. So, it's subjective and inheritance laws don't play any role here. He failed his duties as a son and prince by being away in the darkest hour, so he has no right to be a king. Rights and duties are subjective too.

Reply:
> Though he is not worthy to be the next king

Ok

> So, it's subjective and inheritance laws don't play any role here.

Then everyone, including any chaotic and arbitrary character also have a well-developed sense of justice, since their justice is *just* subjective.

> He failed his duties as a son and prince by being away in the darkest hour, so he has no right to be a king.

Are elves have a law that forbids crown prince to become a king if he fails his duties? He didn't lose any rights.

<details>
<summary>No - book</summary>

![image](https://user-images.githubusercontent.com/46576860/75061747-64e88400-54f2-11ea-864a-502766d527e6.png)

</details>

Reply:
> Then everyone, including any chaotic and arbitrary character also have a well-developed sense of justice, since their justice is _just_ subjective.

Of course, they may have.

> Are elves have a law that forbids crown prince to become a king if he fails his duties? He didn't lose any rights.

Again, it's sense of justice. It's about Kael'thas, not about laws.

Reply:
> Again, it's sense of justice. It's about Kael'thas, not about laws.

If character don't base his "sense of justice" on something external from his feeling and opinions, that tells him "this is right" and "this is wrong", then there is no difference between justice and whims.

> Of course, they may have.

Give me your definition of justice, since I can argue that with the following this logic we can simple delete both arbitrary and just trait from the game.

Reply:
**"All he feels now is hatred for the undead that ravaged his homeland, rage for the Alliance he feels failed his people,[8] desperation over his people's collective addiction to magic, and grief for his ravaged kingdom**

**"Kael'thas was a cunning and powerful mage, as well as a gifted tactician and an accomplished warrior. He was hailed as a true champion of the blood elf people and a defender of their race's ancient heritage**
**Proud and astute, the crown prince of Quel'Thalas could be both an amiable and a ruthless man. He proved himself approachable, but he also advocated drastic measures (such as the extermination of the lethargic orcs) that were at odds with the Kirin Tor's ethics.[12] The prince was imposing to some and charming to others, and almost always regal and composed**. **The Third War's brutal aftermath had a huge effect on Kael, who would assume the mantle of leadership in his murdered father's place and find himself having to deal with the many problems his equally beleaguered people were faced with. Overcome with hatred for the Scourge, on the receiving end of racism from the Alliance resistance, and balancing the weight of a nation on shoulders weary from the addiction ravaging his people, Kael had become a much more desperate leader by the time he left Azeroth for Outland.** 
 **Kael was characterized as having a thorough love for his homeland and its people; but also like Arthas, the extremes he went to for them destroyed most of what he was in the process**. 
**Kael loved his father, Anasterian, but the two did not always see eye-to-eye. While Kael felt loyalty and duty-bound to the Alliance,[43] his father rescinded his own allegiance to it, and recalled the elves to their enchanted kingdom after the Second War. Kael remained in Dalaran despite this, and his father worried whether Kael'thas truly understood that it would one day fall to him to rule Quel'Thalas.[44] Kael was left shattered by Anasterian's death, and strove to make his father's memory proud.[17]"**
[https://wowwiki.fandom.com/wiki/Kael%27thas_Sunstrider]

**"Elor bindel felallan morin’aminor ("Sleep forever in quiet serenity"). I knew this day would come... but I never dreamed it would come so soon. I fear I am not ready, Father. You are the king. You will always be the king. I only ever wanted to make you proud. Grant me the strength to be the man you hoped I would be. Grant me the strength to carry us through this time of despair. Grant me the strength now to lead our people true. Elu’meniel mal alann ("May peace calm your heart")."**

I feel like giving him Just wouldn't make sense that not even Arthas has that trait(until the Lich King Merger event where he can gain it)maybe replace it with diligent?

Reply:
> If character don't base his "sense of justice" on something external from his feeling and opinions, that tells him "this is right" and "this is wrong", then there is no difference between justice and whims.

It's subjective. There's nothing except opinions. You can't say what is wrong and what is bad following only logic because you need to define what means wrong and what means bad first.
https://en.wikipedia.org/wiki/Is%E2%80%93ought_problem

> Give me your definition of justice, since I can argue that with the following this logic we can simple delete both arbitrary and just trait from the game.

Justice is a subjective sense that tells you what is wrong and what is bad. Game-wise... let's say just characters say something about justice and duties, arbitrary - not. Basically, there's no difference and it's a gameplay thing.

Reply:
Ok, it went too far, so let's just resolve it.

Reply:
lmao

Reply:
> I feel like giving him Just wouldn't make sense that not even Arthas has that trait(until the Lich King Merger event where he can gain it)maybe replace it with diligent?

Hmm, true. Kael was kinda diligent. I'll add it.

Well, Arthas cared about duties and justice... till Stratholme I think.
Kael stopped carrying when he betrayed Illidan.

Reply:
I'll replace `stubborn` with `diligent` because he was in abusive relationships with Garithos.

> But, senpai, we are warriors, not--

https://wow.gamepedia.com/Misconceptions_(WC3_BloodElf)